src/components/nav-drawer/themes/shared/header-item/header-item.fluent.scss

```diff
@@ -0,0 +1,9 @@
+@use 'styles/utilities' as *;
+@use '../../light/themes' as *;
+
+$theme: $material;
```

Comment:
it's fluent, but you don't even need this variable as you're not getting anything from the theme.